Application/Education/Certificate/Generator/Service/DataCertificate/IDataFELS.php

```diff
@@ -0,0 +1,278 @@
+<?php
+namespace SPHERE\Application\Education\Certificate\Generator\Service\DataCertificate;
+
+use SPHERE\Application\Education\Certificate\Generator\Service\Data;
+use SPHERE\Application\Education\Lesson\Division\Division;
+use SPHERE\Application\Platform\Gatekeeper\Authorization\Consumer\Consumer;
+use SPHERE\Application\Platform\Gatekeeper\Authorization\Consumer\Service\Entity\TblConsumer;
+
+class IDataFELS
+{
+
+    public static function setCertificateIndividually(Data $Data)
+    {
+
+        $tblConsumerCertificate = Consumer::useService()->getConsumerByAcronym('FELS');
+        if ($tblConsumerCertificate){
+            self::setMsHjInfo($Data, $tblConsumerCertificate);
+            self::setMsJ($Data, $tblConsumerCertificate);
+            self::setGymHjInfo($Data, $tblConsumerCertificate);
+            self::setGymJ($Data, $tblConsumerCertificate);
+        }
+    }
+
+    /**
+     * @param Data        $Data
+     * @param TblConsumer $tblConsumerCertificate
+     */
+    private static function setMsHjInfo(Data $Data, TblConsumer $tblConsumerCertificate)
```

Comment:
Codacy found an issue: [The method setMsHjInfo() has a Cyclomatic Complexity of 15. The configured cyclomatic complexity threshold is 10.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)